### ISSUE TITLE
Can not connect

### ISSUE THREAD

**Issue by balazs-pinter** [flux-web/flux-web#28]
I've installed flux-web with helm. That was successful, but after port-forward, I can not connect to it.

Error:
`kubectl port-forward flux-web-6474867dcb-h7pzm 8080:80
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
Handling connection for 8080
Handling connection for 8080
E1031 17:00:09.234197   86187 portforward.go:400] an error occurred forwarding 8080 -> 80: error forwarding port 80 to pod d5cdeefac5a17cfeea1e06d7f6be803bded35e076901bc0d5a259c3b70e93b43, uid : exit status 1: 2019/10/31 16:00:10 socat[27534] E connect(5, AF=2 127.0.0.1:80, 16): Connection refused`

The flux-web-frontend container doesn't show any logs.

**Comment by idobry** [comment]
Frontend doesn't listen on port 80, try:
'kubectl port-forward NAME 8080'

**Comment by idobry** [comment]
Any update? can I close the this?

**Comment by balazs-pinter** [comment]
Hi, yes, please. Only the help text confused me after the helm install.

**Comment by greenkiwi** [comment]
I was able to connect by forwarding both frontend and backend services.
```
kubectl port-forward --namespace flux s/flux-web-frontend 4080:80
kubectl port-forward --namespace flux s/flux-web-backend 4180:80
```
and then connecting on localhost:4080

I had to update the url to that localhost URL and `wss` to `ws`.
```
---
apiVersion: helm.fluxcd.io/v1
kind: HelmRelease
metadata:
  name: flux-web
  namespace: flux
  annotations:
    fluxcd.io/automated: "false"
    fluxcd.io/tag.chart-image: semver:~4.0
spec:
  releaseName: flux-web
  chart:
    git: git@github.com:flux-web/flux-web.git
    path: chart/flux-web
    ref: master
  values:
    frontend:
        env:
          API_EXTERNAL_URL: //localhost:4180/api/v1
          WS_URL: ws://localhost:4180/ws/v1
          READ_ONLY: "true"
    ingress:
      hosts:
        - host: flux-web.my-domain
          paths: 
            - frontend: /
            - backend: /api
            - backend: /ws
```
Unfortunately this has CORS issue.  I got around that with a chrome extension.

Maybe I can see if I can port forward to the ingress?

**Comment by tmontalbano** [comment]
Hello,

I don't use helm ion my cluster, so, I have generated a template with "helm template" and installed web-flux by applying the generated template in my cluster.

I have also switched the service from ClusterIP to LoadBalancer so I would get an external IP to access it.

Here is the whole thing deployed in the flux namespace:

`➜  ~ kubectl -n flux get all
NAME                           READY   STATUS    RESTARTS   AGE
pod/flux-web-89565489c-rkdmx   2/2     Running   0          16h
pod/flux-web-test-connection   0/1     Error     0          16h

NAME                        TYPE           CLUSTER-IP       EXTERNAL-IP                                                                 PORT(S)        AGE
service/flux-web-backend    LoadBalancer   172.20.233.127   a523f63b5a93c4f39927a4015320ebe5-611659755.ca-central-1.elb.amazonaws.com   80:30786/TCP   16h
service/flux-web-frontend   ClusterIP      172.20.129.185   <none>                                                                      80/TCP         16h

NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/flux-web   1/1     1            1           16h

NAME                                 DESIRED   CURRENT   READY   AGE
replicaset.apps/flux-web-89565489c   1         1         1       16h`

First of all, the flux-web-test-connection pod does not work:

`wget: bad address 'flux-web:80'`

Second, if I redirect both frontend and backend ports like this:

`kubectl port-forward --namespace flux service/flux-web-frontend 4080:80                                                       
Forwarding from 127.0.0.1:4080 -> 8080
Forwarding from [::1]:4080 -> 8080`

`kubectl port-forward --namespace flux service/flux-web-frontend 4180:80
Forwarding from 127.0.0.1:4180 -> 8080
Forwarding from [::1]:4180 -> 8080`

And add the addresses flux-web-frontend and flux-web-backend to my local host file, it won't be able to reach the endpoints. I get a timeout error as if there was a network connectivity issue or something.

Note that `a523f63b5a93c4f39927a4015320ebe5-611659755.ca-central-1.elb.amazonaws.com` points to:
`Non-authoritative answer:
Name:	a523f63b5a93c4f39927a4015320ebe5-611659755.ca-central-1.elb.amazonaws.com
Address: 3.96.25.9
Name:	a523f63b5a93c4f39927a4015320ebe5-611659755.ca-central-1.elb.amazonaws.com
Address: 35.183.146.60`

And here is what I have put in my host file:
`35.183.146.60  flux-web-backend flux flux-web-frontend`

I am sure there is something I'm missing and I would appreciate your help on this matter and be able to test this web ui in our environment.

Thank you!
Thierry.
